src/Flex.svelte

```diff
@@ -3,6 +3,7 @@
   export let align = 'center';
   export let justify = 'center';
   export let reverse = false;
+  export let gap = '0';
```

Comment:
This should be a number, not a string.

```suggestion
  export let gap = 0;
```

Eventually I'd like to add Typescript definitions for this package, which would help make this more clear.

Reply:
Actually, I think this should be the initial value for `gap`, which I think is `'normal normal'` (based on [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/gap)). Either that or we should perhaps leave it undefined/null or something, as I don't want it to be applied if consumers aren't using it.

Reply:
i think that leaving as null / undefined would be better (because it wouldnt be applied)

Reply:
Yes let's leave it as `null` or `undefined`